packages/graphql-testing/README.md

```diff
@@ -0,0 +1,94 @@
+# `@shopify/graphql-testing`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fgraphql-testing.svg)](https://badge.fury.io/js/%40shopify%2Fgraphql-testing.svg)
+
+This utility aid with the following testing scenarios:
+
+1. Testing graphql request with mock data.
+2. Testing the state of your application before / after graphql request(s) resolves.
+
+It does this by creating a mock `ApolloClient` that can be access using `const graphQL = createGraphQLFactory()(mockData); const apolloClient = graphQL.client` and pass into your test setup.
+
+To assert with operations that had been executed. Find list of operations using `apolloClient.operations.all()`.
+
+To resolve all pending graphql request(s). Use `await graphQL.resolveAll()` in your test.
+
```

Comment:
This is done by creating a mock `ApolloClient` that you will pass into your test setup.

```
const graphQL = createGraphQLFactory()(mockData); 
const apolloClient = graphQL.client
```

The `apolloClient.operations.all()` will return a list of operations that you can assert have been executed from within your tests.

To resolve any pending graphQL requests, use `await graphQL.resolveAll()`.

Reply:
On second thought, I think this part gets too far into the Usage, and can be merged with that section below.